.script/utils/gitWrapper.ts

```diff
@@ -29,7 +43,8 @@ export async function GetDiffFiles(fileTypeSuffixes?: string[], filePathFolderPr
 
   let fileTypeSuffixesLogValue = typeof fileTypeSuffixes === "undefined" ? null : fileTypeSuffixes.join(",");
   let filePathFolderPreffixesLogValue = typeof filePathFolderPreffixes === "undefined" ? null : filePathFolderPreffixes.join(",");
-  console.log(`${filterChangedFiles.length} files changed in current PR after filter. File Type Filter: ${fileTypeSuffixesLogValue}, File path Filter: ${filePathFolderPreffixesLogValue}`);
+  let fileKindsLogValue = typeof fileKinds === "undefined" ? null : fileKinds.join(",");
+  console.log(`${filterChangedFiles.length} files changed in current PR after filter. File Type Filter: ${fileTypeSuffixesLogValue}, File path Filter: ${fileKindsLogValue}, File path Filter: ${filePathFolderPreffixesLogValue}`);
```

Comment:
change `File path Filter: ${fileKindsLogValue}` to `File Kind Filter: ${fileKindsLogValue}` ( you forget to change the `path `text..)